data/huion-wh1409.tablet

```diff
@@ -0,0 +1,40 @@
+# HUION
+# WH1409
+#
+# sysinfo.DFvHUfcRjW.tar.gz
+# https://github.com/linuxwacom/wacom-hid-descriptors/issues/385
+#
+# Button Map:
+# (A=1, B=2, C=3, ...)
+#
+#         *----------------------------*
+#    A B  |                            |
+#    c D  |                            |
+#         |                            |
+#    E F  |           TABLET           |
+#    G H  |                            |
+#         |                            |
+#    I J  |                            |
+#    K L  |                            |
+#         *----------------------------*
+
+[Device]
+Name=TABLET Pen Tablet
+ModelName=
+DeviceMatch=usb|256c|006e|TABLET Pen Tablet Pad;usb|256c|006e|TABLET Pen Tablet Pen;usb|256c|006e|TABLET Pen Tablet Touch Strip;usb|256c|006e|TABLET Pen Tablet Dial
```

Comment:
tbh, I'm a bit hesitant to merge this because it's probably going to conflict with other devices that re-use the same PID. As it happens this is the first device with his particular name match but still...

I'm guessing you're using the digimend drivers for this tablet? What's the value of HID_UNIQ for this device, that should contain the firmware string (`udevadm info --export-db` then search for the tablet).